### ISSUE TITLE
[Accessibility][Advisory][Button] Icon text may not be appropriate

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1678]
**Steps to Reproduce**
1. With NVDA operating, navigate to the "Contact Us" button, and note announcement of "email" CSS content.

**Current Behavior**
Text alternatives for icons in buttons are provided through the CSS `content` property, which screen reader users may or may not hear announced. 

Text content in the CSS `content` property is supported by modern Chrome and Firefox browsers and JAWS and NVDA versions, but is not supported in IE with either screen reader. Announcements will vary from one assistive technology/browser combination to another. 

**Expected behavior/code**
Icons that provide meaning have textual alternatives that are consistently announced by screen readers; icons that are decorative are not announced by screen readers.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users

**WCAG SC**
1.1.1

**Code snippet**
```
<span class="cmp-button**icon cmp-button**icon--email"></span>
```

``{{
.t-cmp-clean .cmp-button__icon--email:before {
    content: 'email';
}
}}``

**Possible Solution**
If a text alternative for an icon is necessary to understand the purpose of a button, provide that alternative through a mechanism that has more consistent browser/assistive technology support, such as visually-hidden text. If the icon is purely decorative, it should be hidden from assistive technology using `aria-hidden="true"`.

If one of these options is not possible, ensure that any text alternative for icons added through CSS is a) appropriately descriptive to convey the purpose of the icon (so it does not interfere with understanding for those users who hear it announced) and b) not necessary to understand the purpose of the button (so those users who do not hear it announced will not lose important information).

**Comment by bpauli** [comment]
@adobe export issue to Jira project SITES

**Comment by bpauli** [comment]
done in https://github.com/adobe/aem-core-wcm-components/pull/1703
